tests/test_local_env.py

```diff
@@ -1,13 +1,56 @@
-"""Verify behavior of package module looking at a given venv."""
+"""Verify behavior of package module looking at a given Python environment."""
+import sys
 import venv
 
+import pytest
+
 from fawltydeps.packages import (
     DependenciesMapping,
     LocalPackageLookup,
     Package,
     resolve_dependencies,
 )
 
+major, minor = sys.version_info[:2]
+
+# When the user gives us a --pyenv arg, what are the the possible paths inside
+# that Python environment that they might point at (that we should accept)
+env_subdirs = [
+    "",
+    "bin",
+    "lib",
+    f"lib/python{major}.{minor}",
+    f"lib/python{major}.{minor}/site-packages",
+]
+
+
+@pytest.mark.parametrize(
+    "subdir", [pytest.param(d, id=f"venv:{d}") for d in env_subdirs]
+)
+def test_determine_package_dir__various_paths_in_venv(tmp_path, subdir):
+    venv.create(tmp_path, with_pip=False)
+    path = tmp_path / subdir
+    expect = tmp_path / f"lib/python{major}.{minor}/site-packages"
```

Comment:
Since it's used a few places, maybe a helper could be added that captures the logic of "give me the path to `site-packages` if i give you the base path, the major version, and the minor version"